solr/core/src/java/org/apache/solr/handler/tagger/XmlOffsetCorrector.java

```diff
@@ -45,21 +43,12 @@
  */
 public class XmlOffsetCorrector extends OffsetCorrector {
 
-  //TODO use StAX without hard requirement on woodstox.   xmlStreamReader.getLocation().getCharacterOffset()
+  //TODO use StAX without hard requirement on woodstox. xmlStreamReader.getLocation().getCharacterOffset()
 
-  private static final XMLInputFactory2 XML_INPUT_FACTORY;
+  private static final XMLInputFactory XML_INPUT_FACTORY;
   static {
-    // note: similar code in Solr's EmptyEntityResolver
-    XML_INPUT_FACTORY = new WstxInputFactory();
-    XML_INPUT_FACTORY.setXMLResolver(new XMLResolver() {
-      @Override
-      public InputStream resolveEntity(String publicId, String systemId, String baseURI, String namespace) {
-        return ClosedInputStream.CLOSED_INPUT_STREAM;
-      }
-    });
-    // TODO disable DTD?
```

Comment:
Is this TODO still applicable? Is this now disabled by default? If not, leave the TODO comment in.